web/src/main/java/com/microsoft/applicationinsights/web/internal/cookies/SessionCookie.java

```diff
@@ -22,8 +22,7 @@
 package com.microsoft.applicationinsights.web.internal.cookies;
 
 import java.text.ParseException;
-import java.util.Calendar;
-import java.util.Date;
+import java.util.*;
```

Comment:
Please do not use * imports. Edit it to add individual import statements,

Reply:
fixed